plottable.d.ts

```diff
@@ -4087,7 +3968,6 @@ declare module Plottable {
         }
         class Hover extends Interaction.AbstractInteraction {
             _componentToListenTo: Hoverable;
-            constructor();
             _anchor(component: Hoverable, hitBox: D3.Selection): void;
             /**
              * Attaches an callback to be called when the user mouses over an element.
```

Comment:
merging develop should actually have these changes disapppear... Let me take a look locally